benchmark.go

```diff
@@ -112,13 +209,27 @@ func runBenchmark(out output, logger *zap.Logger, allOpts Options, resolved reso
 
 	goMaxProcs := opts.setGoMaxProcs()
 	numConns := opts.getNumConnections(goMaxProcs)
-	out.Printf("Benchmark parameters:\n")
-	out.Printf("  CPUs:            %v\n", goMaxProcs)
-	out.Printf("  Connections:     %v\n", numConns)
-	out.Printf("  Concurrency:     %v\n", opts.Concurrency)
-	out.Printf("  Max requests:    %v\n", opts.MaxRequests)
-	out.Printf("  Max duration:    %v\n", opts.MaxDuration)
-	out.Printf("  Max RPS:         %v\n", opts.RPS)
+
+	benchmarkParameters := BenchmarkParameters{
+		CPUs:        goMaxProcs,
+		Connections: numConns,
+		Concurrency: opts.Concurrency,
+		MaxRequests: opts.MaxRequests,
+		MaxDuration: opts.MaxDuration.String(),
+		MaxRPS:      opts.RPS,
+	}
+
+	formatAsJSON := false
+	if opts.Format == "json" || opts.Format == "JSON" {
```

Comment:
alternatively, `if strings.ToLower(opts.Format) == "json"` which also will allow `Json`

Reply:
+1, this may worth doing, if we do, we should change the output error message to state the its case-insensitive, maybe `please specify 'json' (case-insensitive) ...`

Reply:
OK with keeping the error simple, and just saying "please specify 'json'", we just want to support other cases to follow the [robustness principle](https://en.wikipedia.org/wiki/Robustness_principle),
> Be conservative in what you send, be liberal in what you accept